docs/source/jit.rst

```diff
@@ -1097,6 +1105,10 @@ check the correctness of the model as you go.
 
 .. autofunction:: ignore
 
+.. autofunction:: unused
```

Comment:
The code example in docstring for `unused`  should have a `.. testcode::` directive on it, see the other documented functions for examples

Reply:
it already does

Reply:
Sorry I meant `is_scripting` above, `unused` looks fine